sites/svelte.dev/src/routes/docs/+layout.svelte

```diff
@@ -42,6 +48,14 @@
 		all: unset;
 	}
 
+	.category {
+		font: 700 var(--sk-text-s) var(--sk-font);
+		text-transform: uppercase;
+		letter-spacing: 0.12em;
+		margin: 0;
```

Comment:
i feel like it needs a little margin between the two headings
```suggestion
		margin: 0 0 0.5em;
```